<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/newPrefix/Windows (2).png)</string>
  </property>
  <widget class="QPushButton" name="Type2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Type1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Type3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Type4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../code/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
